src-web/definitions/application.js

```diff
@@ -238,17 +238,16 @@ export default {
   moduleKeys: {
     title: 'page.componentView.title',
     headers: [
-      {key: 'status', header: msgs.get('table.header.status')},
-      {key: 'name', header: msgs.get('table.header.name')},
-      {key: 'compositeKind', header: msgs.get('table.header.kind')},
-      {key: 'namespace', header: msgs.get('table.header.namespace')},
-      {key: 'platform', header: msgs.get('table.header.platform')},
-      {key: 'menuAction', header: msgs.get('table.header.action')},
-      {key: 'title', header: 'Title'},
-      {key: 'description', header: 'Description'},
-      {key: 'section_data', header: 'section_data'},
-      {key: 'section_map', header: 'section_map'}
-
+      {key: 'status', header: msgs.get('table.header.status'), type: 'status'},
```

Comment:
Make types constants that live inside of the ResourceTable.  
export const SEARCH_HEADER_TYPES= {
  NOT_SEARCHABLE: 'NOT_SEARCHABLE',
   ...
}